src/WebJobs.Extensions.DurableTask/EndToEndTraceHelper.cs

```diff
@@ -252,7 +253,58 @@ public static string LocalSlotName
 
             this.logger.LogInformation(
                 "{instanceId}: Function '{functionName} ({functionType})', version '{version}' received a '{eventName}' event. State: {state}. HubName: {hubName}. AppName: {appName}. SlotName: {slotName}. ExtensionVersion: {extensionVersion}. SequenceNumber: {sequenceNumber}.",
-                instanceId, functionName, functionType, version, eventName, FunctionState.ExternalEventRaised, hubName, LocalAppName, LocalSlotName, ExtensionVersion, this.sequenceNumber++);
+                instanceId, functionName, functionType, version, eventName, FunctionState.ExternalEventRaised, hubName,
+                LocalAppName, LocalSlotName, ExtensionVersion, this.sequenceNumber++);
+        }
+
+        public void EventGridMessageSent(
```

Comment:
I'm a little confused. The method name is `EventGridMessageSent` but the trace messages are reporting errors. it seems to me that there needs to be two different traces methods: one for successfully sent messages (Informational) and one for errors.

Reply:
Hi @cgillum, I've done the review. The CI was broken. The error message is related the Storage Account conflict. At least, when I run the test locally, the final commit is all passed.